### DISCUSSION TITLE
调整了一下说说的前端样式

### DISCUSSION THREAD

**Discussion by Uyoahz26** [Qexo/Qexo#253] [Ideas]
在 [Guo Le @夜的第八章](https://guole.fun/)修改的基础上再调整了一下样式，[预览](https://uyoahz.cn/daodao)，[使用方法](https://uyoahz.cn/2022111530055/)，作者大大可以瞅一瞅，仓库地址[daodao](https://github.com/Uyoahz26/daodao)
![1669691108294](https://user-images.githubusercontent.com/42881988/204428745-69ec43a0-11b0-4c24-bbe8-0fea4327675c.png)

**Comment by Fgaoxing** [comment]
每個人都可以自己編寫！！



在 2022-11-29 11:05:41，"Uyoahz" ***@***.***> 写道：

在 Guo Le @夜的第八章修改的基础上再调整了一下样式，预览，使用方法，作者大大可以瞅一瞅

**Comment by Fgaoxing** [comment]
看起來很不錯



在 2022-11-29 11:05:41，"Uyoahz" ***@***.***> 写道：

在 Guo Le @夜的第八章修改的基础上再调整了一下样式，预览，使用方法，作者大大可以瞅一瞅

**Comment by am-abudu** [comment]
看起来很棒，可以直接给qexo/Qstatic进行pr（添加css/js）

**Reply by am-abudu** [reply]
另外，现在说说已经支持了自定义的参数，你可以使用自定义参数进行个性化！
